src/components/SidebarGroup/SidebarGroupComponent/SidebarGroupComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Selected } from '../../../stories/Box.stories'
+import checkmark from './Checkmark.png'
+
+export interface SidebarGroupComponentProps {
+    //Name of the component.
+    componentName: string;
+
+    //Is the state completed?
+    completed?: boolean;
+
+    //Is the state selected (being done)?
+    selected?: boolean;
+}
+
+const SidebarGroupComponent: React.FC<SidebarGroupComponentProps> = ({ componentName, completed, selected }) => {
+    
+    {/*TODO: Need to figure out a way to round only the top and bottom components */}
```

Comment:
Idea: You can add a "first" / "last" flag (boolean variables) as a prop when you use `items.map` to iterate through the available items, then conditionally apply inline styles to add border-radius values to two of the corners.
https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius